src/plugins/intel_cpu/src/nodes/executors/acl/acl_weights.cpp

```diff
@@ -0,0 +1,298 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "acl_fullyconnected.hpp"
+#include "acl_utils.hpp"
+#include "nodes/common/cpu_convert.h"
+#include "nodes/executors/executor.hpp"
+#include "nodes/executors/memory_arguments.hpp"
+#include "memory_desc/cpu_memory_desc_utils.h"
+#include "utils/debug_capabilities.h"
+
+namespace ov {
+namespace intel_cpu {
+
+VectorDims acl_fc_executor::makeDummyInputDims(const Shape& inShape, const Shape& wShape) {
```

Comment:
Regarding the name of the file.
From my understanding, the functions in this file are directly related to fullyconnected operation and also includes some functions such as makeDummyInputDims which are not completely about weights.
Can we rename it to something like acl_fullyconnected_utils.cpp
And the header as well.

Reply:
Renamed it to `acl_fullyconnected_utils`